python/lsst/ts/m2gui/utils.py

```diff
@@ -222,3 +224,58 @@ def get_num_actuator_ring(ring):
         return 18
     else:
         raise ValueError(f"Not supported ring: {ring!r}")
+
+
+def prompt_dialog_warning(description, is_prompted=True):
+    """Prompt the warning dialog.
```

Comment:
Shows warning dialog.

Reply:
Done in the commit: [b2dae7c](https://github.com/lsst-ts/ts_m2gui/pull/10/commits/b2dae7c31b0f66668b2974b6761c5d7795e35f4b). Thanks!